### PR TITLE
CareTeam[r4]:remove encounter scope

### PR DESCRIPTION
**Pull request by ramya1096**
Description
removed encounter scope for careteams
Screenshot 2021-04-12 at 1 21 45 PM

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/care-provision/care-team.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Search for CareTeams that meet supplied query parameters:
 
 _Implementation Notes_
 
-* For CareTeam Search operations, both Careteam.read and Encounter.read scopes are required.
+* For CareTeam Read operations, only Careteam.read is required.
```
Comment (aki-mathur) [inline content/millennium/r4/care-provision/care-team.md:38]: It should be `For CareTeam Search operations`

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: content/millennium/r4/care-provision/care-team.md (hunk 1 above)
Comment (ramya1096) [reply content/millennium/r4/care-provision/care-team.md]: changes addressed

**Review by aki-mathur — APPROVED** [review]

File: content/millennium/r4/care-provision/care-team.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Search for CareTeams that meet supplied query parameters:
 
 _Implementation Notes_
 
-* For CareTeam Search operations, both Careteam.read and Encounter.read scopes are required.
+* For CareTeam Search operations, only Careteam.read is required.
```
Comment (NimishaMN) [inline content/millennium/r4/care-provision/care-team.md:38]: @ramya1096 , can we remove the complete Implementation Notes section, as we had added them to mention Encounter.read scope is required?

cc: @UjwalaBheema , @ashu-bdvt

Comment (ramya1096) [reply content/millennium/r4/care-provision/care-team.md]: i have removed implementation notes section after confirming with ujwala.

**Review by UjwalaBheema — APPROVED** [review]

**Review by NimishaMN — APPROVED** [review]

**Comment by Zonglin123** [comment]
Screenshot after changes published.
Screen Shot 2021-04-15 at 5 49 40 PM

**Pull request merged**